ancestrymanager/ancestrymanager.go

```diff
@@ -35,15 +37,14 @@ type manager struct {
 // as value to the offline cache. If the key is not prefix with `projects/` or
 // `folders/`, it will be considered as a project. If no resourceManager is
 // provided, API requests for ancestry will be disabled.
-func New(resourceManager *cloudresourcemanager.Service, entries map[string]string, errorLogger *zap.Logger) (AncestryManager, error) {
-	return newManager(resourceManager, entries, errorLogger)
-}
-
-func newManager(resourceManager *cloudresourcemanager.Service, entries map[string]string, errorLogger *zap.Logger) (*manager, error) {
+func New(cfg *resources.Config, offline bool, entries map[string]string, errorLogger *zap.Logger) (AncestryManager, error) {
```

Comment:
I am fine with us going back to taking cfg/offline as args now that we are going to have better tools for customizing the cfg's user agent and http client 👍 

Could you update the comment above to reflect this change?

Reply:
Done.